make.js

```diff
@@ -386,6 +390,20 @@ function categorizeCalls(apis) {
     }
 }
 
+function setPrerequisiteCalls(apis) {
+    for (var i = 0; i < apis.length; i++) {
+        var api = apis[i];
+        prereqs = [];
+
+        if (api.name == "Groups") {
+            prereqs.push(api.calls.find(elem => elem.name === "CreateGroup"))

```

Comment:
This will end up being an ordered list for whatever calls need to happen before the bulk of the tests are ran. We can just break this out into a json file or whatever if we want later